keras_retinanet/preprocessing/generator.py

```diff
@@ -234,7 +234,7 @@ def compute_input_output(self, group):
         # compute network targets
         targets = self.compute_targets(image_group, annotations_group)
 
-        return inputs, targets
+        return inputs, targets, annotations_group
```

Comment:
I think we should be careful with the generator API. Returning the annotations here seems like a bit of a hack. I'd like to discuss this at the office a bit.